earthaccess/auth.py

```diff
@@ -93,7 +92,6 @@ def login(
                 * **"environment"**:
                     Retrieve a username and password from $EARTHDATA_USERNAME and $EARTHDATA_PASSWORD.
             persist: Will persist credentials in a `.netrc` file.
-            system (Env): the EDL endpoint to log in to Earthdata, defaults to PROD
```

Comment:
We only want to remove the incorrectly-documented type here. Should be `system: the EDL endpoint to log in to Earthdata, defaults to PROD`

See the type signature above, the type is actually `Optional[System]`:

```python
 system: Optional[System] = None,
```

The documentation generator is capable of reading this, so we don't need to also mention the type in the docstring!